### PR TITLE
Add propose/vote p2p message latency to traffic stats

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add propose/vote p2p message latency to traffic stats
- Remove traffic stats manager from BlockchainNode constructor

### CODE REVIEW

File: consensus/index.js [hunk 1]
```diff
@@ -220,6 +222,20 @@ class Consensus {
     return true;
   }
 
+  addTrafficEventsForProposalTx(proposalTx) {
```
Comment (liayoo) [inline consensus/index.js:225]: could you move these to the consensus-util.js?

File: common/constants.js [hunk 2]
```diff
@@ -569,6 +569,8 @@ const TrafficEventTypes = {
   PROPOSE_BEFORE_BLOCK: 'propose_before_block',
```
Comment (liayoo) [inline common/constants.js:569]: minor) `// Latency` -> `// Consensus messages`?

**Review by liayoo — APPROVED** [review]
LGTM with some minor comments. Thanks!

**Review by minsulee2 — APPROVED** [review]
LGTM!

File: common/constants.js (hunk 2 above)
Comment (platfowner) [reply common/constants.js:569]: Just changed to 'Consensus & block producing'. PTAL~

File: consensus/index.js (hunk 1 above)
Comment (platfowner) [reply consensus/index.js]: Done, thanks!

**Pull request merged**